doc/classes/ParallaxLayer.xml

```diff
@@ -4,7 +4,8 @@
 		A parallax scrolling layer to be used with [ParallaxBackground].
 	</brief_description>
 	<description>
-		A ParallaxLayer must be the child of a [ParallaxBackground] node. All child nodes will be affected by the parallax scrolling of this layer.
+		A [code]ParallaxLayer[/code] must be the child of a [ParallaxBackground] node. Each [code]ParallaxLayer[/code] can be set to move at different speeds relative to the camera movement or the [member ParallaxBackground.scroll_offset] value.
```

Comment:
- Remove [code] tags around class name
- as above:

Each [ParallaxLayer] can move at a different speed using the [member ParallaxBackground.scroll_offset] value. This creates an illusion of depth in a 2D game. If not used with a [Camera2D], you must manually calculate the [member scroll_offset].